Missionframework/onPlayerRespawn.sqf

```diff
@@ -16,3 +18,28 @@ if (isNil "GRLIB_respawn_loadout") then {
 };
 
 [] call KPLIB_fnc_addActionsPlayer;
+
+// Support Module handling
+if ([
+    false,
+    player isEqualTo ([] call KPLIB_fnc_getCommander),
+    (getPlayerUID player) in KP_liberation_suppMod_whitelist,
+    true
+] select KP_liberation_suppMod) then {
+    waitUntil {!isNil "KPLIB_suppMod_req" && !isNil "KPLIB_suppMod_arty" && time > 5};
+
+    // Remove link to corpse, if respawned
+    if (!isNull _oldUnit) then {
+        KPLIB_suppMod_req synchronizeObjectsRemove [_oldUnit];
+        _oldUnit synchronizeObjectsRemove [KPLIB_suppMod_req];
```

Comment:
Isn't the synchronization two way? I don't think you need to link/unlink both sides.

Reply:
Yeah, normally it is. I've also just done this one way for the HC implementation.
I used this in both directions at this place, as the `BIS_fnc_addSupportLink` does it also internally. So a "shouldn't be necessary, but doesn't hurt and as BIS use it in their function, better safe then sorry"

Reply:
👍